src/apple/cocoapod.ts

```diff
@@ -40,21 +41,31 @@ export async function addCocoaPods(projPath: string): Promise<boolean> {
     podContent.slice(insertIndex);
   fs.writeFileSync(podfile, newFileContent, 'utf8');
 
-  const loginSpinner = clack.spinner();
-
   clack.log.step('Sentry pod added to the project podFile.');
-  loginSpinner.start("Running 'pod install'. This may take a few minutes...");
+
+  await podInstall();
+
+  return true;
+}
+
+export async function podInstall(dir = '.') {
+  const installSpinner = clack.spinner();
+  installSpinner.start("Running 'pod install'. This may take a few minutes...");
 
   try {
-    await bash.execute('pod install --silent');
-    loginSpinner.stop('Running "pod install"');
+    await bash.execute(`cd ${dir} && pod repo update`);
+    await bash.execute(`cd ${dir} && pod install --silent`);
+    installSpinner.stop('Pods installed.');
```

Comment:
l/q: do we need to `cd` back to the previous dir or are we fine?

Reply:
we don't have to, it changes pwd only for the current command,
thanks for the question, that could be an annoying bug if it would change the pwd of the wizard process